app/lib/screens/wallets/send.dart

```diff
@@ -176,131 +179,151 @@ class _WalletSendScreenState extends State<WalletSendScreen> {
-                        color: Theme.of(context).colorScheme.onSurface,
-                      ),
-                  controller: fromController,
-                  decoration: InputDecoration(
-                    labelText: 'From (name: ${widget.wallet.name})',
-                  )),
-            ),
-            const SizedBox(height: 10),
-            ListTile(
-              title: TextField(
-                  style: Theme.of(context).textTheme.bodyMedium!.copyWith(
-                        color: Theme.of(context).colorScheme.onSurface,
-                      ),
-                  controller: toController,
-                  decoration: InputDecoration(
-                      labelText: 'To',
-                      errorText: toAddressError,
-                      suffixIcon: IconButton(
-                          onPressed: () {
-                            Navigator.of(context).push(MaterialPageRoute(
-                              builder: (context) => ContractsScreen(
-                                  chainType: chainType,
-                                  currentWalletAddress: fromController.text,
-                                  wallets: widget.allWallets,
-                                  onSelectToAddress: _selectToAddress),
-                            ));
-                          },
-                          icon: const Icon(Icons.person)))),
-            ),
-            const SizedBox(height: 10),
-            ListTile(
-              title: TextField(
-                  style: Theme.of(context).textTheme.bodyMedium!.copyWith(
-                        color: Theme.of(context).colorScheme.onSurface,
-                      ),
-                  keyboardType:
-                      const TextInputType.numberWithOptions(decimal: true),
-                  controller: amountController,
-                  decoration: InputDecoration(
-                      labelText: 'Amount (Balance: $balance)',
-                      hintText: '100',
-                      suffixText: 'TFT',
-                      errorText: amountError)),
-              subtitle: Text(
-                  'Max Fee: ${chainType == ChainType.Stellar ? 0.1 : 0.01} TFT'),
-            ),
-            const SizedBox(height: 10),
-            if (chainType == ChainType.Stellar)
-              ListTile(
-                title: TextField(
-                    style: Theme.of(context).textTheme.bodyMedium!.copyWith(
-                          color: Theme.of(context).colorScheme.onSurface,
+                  const SizedBox(height: 10),
+                  ListTile(
+                    title: TextField(
+                        style: Theme.of(context).textTheme.bodyMedium!.copyWith(
+                              color: Theme.of(context).colorScheme.onSurface,
+                            ),
+                        focusNode: textFieldFocusNode,
+                        keyboardType: TextInputType.number,

```

Comment:
why do we remove the decimal?

`TextInputType.numberWithOptions(decimal: true)`